addons/maintenance/models/maintenance.py

```diff
@@ -306,6 +306,13 @@ def _get_default_team_id(self):
     maintenance_team_id = fields.Many2one('maintenance.team', string='Team', required=True, default=_get_default_team_id, check_company=True)
     duration = fields.Float(help="Duration in hours.")
     done = fields.Boolean(related='stage_id.done')
+    instruction_type = fields.Selection([
+        ('pdf', 'PDF'), ('google_slide', 'Google Slide'), ('text', 'Text')],
+        string="Instruction", default="text"
```

Comment:
Since you also have the 'worksheet template' above, I prefer to use different terminology for these 2 concepts. 
We could however delete the labels alltogether as such : https://tinyurl.com/ysb4w7ct